Slim/Player/Player.pm

```diff
@@ -109,8 +109,8 @@ sub init {
 	Slim::Buttons::Home::updateMenu($client);
 
 	# fire it up!
+	$client->power($prefs->client($client)->get('power'));
```

Comment:
Isn't this still a no-op, since currOn == on here?

Reply:
I guess it's not in case `Slim:vButtons::Common::mode` returns `undef` ... not sure about the conditions this is happening under though.

Reply:
It's the case when player just join and if you don't do power() then the following display init ends up with an assert()